src_assets/common/assets/web/configs/tabs/encoders/NvidiaNvencEncoder.vue

```diff
@@ -114,6 +114,26 @@ const config = ref(props.config)
               </select>
               <div class="form-text">{{ $t('config.nvenc_h264_cavlc_desc') }}</div>
             </div>
+
+            <!-- Force disable V-Sync (NVAPI) -->
+            <div class="mb-3" v-if="platform === 'windows'">
```

Comment:
This one is fine, but I'm not sure why the one above is missing the conditional for the windows platform... or if any of them are even needed as the entire nvenc encoder is Windows only.